default/scripting/species/SP_TRITH.focs.txt

```diff
@@ -6,7 +6,7 @@ Species
     CanProduceShips
     CanColonize
 
-    tags = [ "SELF_SUSTAINING" "TELEPATHIC" "XENOPHOBIC" "BAD_INFLUENCE" "GOOD_SHIP_SHIELDS" "GREAT_HAPPINESS" "AVERAGE_SUPPLY" "GOOD_DETECTION" "GOOD_STEALTH" "PEDIA_SELF_SUSTAINING_SPECIES_CLASS" "PEDIA_XENOPHOBIC_SPECIES_TITLE" ]
+    tags = [ "SELF_SUSTAINING" "TELEPATHIC" "XENOPHOBIC" "GREAT_INFLUENCE" "GOOD_SHIP_SHIELDS" "BAD_HAPPINESS" "AVERAGE_SUPPLY" "GOOD_DETECTION" "GOOD_STEALTH" "PEDIA_SELF_SUSTAINING_SPECIES_CLASS" "PEDIA_XENOPHOBIC_SPECIES_TITLE" ]
```

Comment:
hello bad_happiness tag

Reply:
oh, its in trith (which has the correct macro). sorry for the interruption.

Reply:
Yeah, you are being very helpful xD